lib/userAgent.js

```diff
@@ -27,6 +27,8 @@ var MOBILE_OS_FAMILIES = {
 
 var ELLIPSIS = '\u2026'
 
+const OLD_FIREFOX_IOS_USER_AGENT = /^Firefox-iOS-FxA/
```

Comment:
> ...this is not an "old" UA.

Oh, I didn't realise that, sorry. Will fix.

> ...maybe that is why we are getting Android "tablets" in the initial device view.

I think you're on to something here, testing it out now.